src/docs/cookbook/forms/focus.md

```diff
@@ -9,22 +9,24 @@ next:
 ---
 
 When a text field is selected and accepting input, it is said to have "focus."
-Generally, users can focus text fields by tapping on them, and developers
-can focus text fields using the tools described in this recipe.
+Generally, users shift focus to a text field by tapping,
+and developers shift focus to a text field programmatically by
+using the tools described in this recipe.
 
 Managing focus is a fundamental tool for creating forms with an intuitive
-flow. For example, say we have a search screen with a text field. When
-the user navigates to the search screen, we can focus the search term text field.
+flow. For example, say you have a search screen with a text field.
+When the user navigates to the search screen,
+you can focus on the search term text field.
```

Comment:
"focus on" isn't right here. Maybe "set the focus to the text field for the search term."